common/djangoapps/third_party_auth/pipeline.py

```diff
@@ -150,6 +151,7 @@ def is_api(auth_entry):
 
     AUTH_ENTRY_LOGIN_API,
     AUTH_ENTRY_REGISTER_API,
+    AUTH_ENTRY_SUDO
```

Comment:
@waheedahmed Yes you can, though I'm not a big fan of modifying an enum definition at runtime. Personally I'd much rather see you define the string constant in this file as is done for all the other constants in this enum. `AUTH_ENTRY_SUDO` is just a constant and its actual value doesn't matter, so there's no need to import the django_sudo_helpers app in order to define the constant, and there's no harm in having it always defined. (The pipeline case we had discussed is somewhat different because there you are injecting a code dependency.)

If you do take the runtime modification approach, please add a comment explaining it.